Formula/gulp-cli.rb

```diff
@@ -0,0 +1,33 @@
+require "language/node"
+
+class GulpCli < Formula
+  desc "Command-line utility for Gulp"
+  homepage "https://github.com/gulpjs/gulp-cli"
+  url "https://registry.npmjs.org/gulp-cli/-/gulp-cli-2.3.0.tgz"
+  sha256 "0a5a76e5be9856edf019fb5be0ed8501a8d815da1beeb9c6effca07a93873ba4"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "npm", "init", "-y"
+    system "npm", "install", *Language::Node.local_npm_install_args, "gulp"
```

Comment:
Why does it need to install gulp again?

Reply:
Found a good explanation: https://stackoverflow.com/a/35573218

Reply:
Okay, that's a weird system but sure.